source/releasenotes/2024-01-24-wp-cli-v2-9-0-release-now-available-on-pantheon.md

```diff
@@ -0,0 +1,21 @@
+---
+title: WP-CLI v2.9.0 release now available on Pantheon
+published_date: "2024-01-25"
+categories: [wordpress]
+---
+
+We're thrilled to announce the availability of WP-CLI v2.9.0 on the Pantheon platform.
+
+WP-CLI is easily accessible for any WordPress site on Pantheon through our CLI, [Terminus](https://docs.pantheon.io/terminus). Manage your WordPress installations efficiently with the power of WP-CLI at your fingertips.
```

Comment:
@IngridKwok Can you replace all instances of absolute links like this from your entries today? We want relative links, so in this case it would be `[Terminus](/terminus)`

If you use absolute link for a relative page, the behavior changes. Absolute links will open in a new window which should only happen when linking outside of the docs.pantheon.io site